Jenkinsfile

```
@@ -1,3 +1,5 @@
+@Library('pipeline-library@code-coverage')
```

weird groovy thing means you need to add an _ if it's not in a class I think
```suggestion
@Library('pipeline-library@code-coverage') _
```